manifests/params.pp

```diff
@@ -303,17 +303,11 @@
       $unavailable_runmodes = []
     }
     'Redhat' : {
-      # PSBM is a CentOS 6 based distribution
-      # it reports its $osreleasemajor as 2, not 6.
-      # thats why we're matching for '2' in both parts
-      # Amazon Linux is like RHEL6 but reports its osreleasemajor as 2017 or 2018.
       $agent_restart_command = $facts['os']['release']['major'] ? {
```

Comment:
I'd go a step further and assume systemd (once EL6 has been dropped). That means I'd just remove the whole case statement and reduce it to `$agent_restart_command = "/usr/bin/systemctl reload-or-restart ${service_name}"`